### ISSUE TITLE
Error compiling sass

### ISSUE THREAD

**Issue by matthewfowles** [Compass/compass#1508]
Hi 

I'm using yeoman and grunt. When i run to compile sass to css using compass get syntax error anywhere the are 3 dots in the sass.

Error: 

``` css
Syntax error: Invalid CSS after \"...-shadow($shadow\": expected \")\", was \"...) {\"\A on line 121 of /Users/matthewfowles/Desktop/maps/app/bower_components/sass-bootstrap/lib/_mixins.scss\A from line 3 of /Users/matthewfowles/Desktop/maps/app/bower_components/sass-bootstrap/lib/bootstrap.scss\A from line 3 of /Users/matthewfowles/Desktop/maps/app/styles/main.scss

```

Any help greatly appreciated

**Comment by scottdavis** [comment]
can you gist the scss file?

**Comment by Asken** [comment]
If you haven't already solved it, the dots are introduced in sass 3.2.
